src/main/java/helpedfunctions/StringiusUtils.java

```diff
@@ -0,0 +1,126 @@
+package helpedfunctions;
+
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class StringiusUtils {
+    private static final Logger LOGGER = LogManager.getLogger(StringiusUtils.class);
+
+    public static boolean isEmailValid(String email) {
+        return !isEmailEmpty(email) && isEmailContainEt(email) && isEmailIsNotStartsFromDot(email) && isEmailDomainLengthValid(email)
+                && isEmailDomainFromDotLengthValid(email) && isEmailLastCharValid(email) && isEmailLengthValid(email)
+                && isEmailLocalPartLengthValid(email);
+    }
+
+    private static String getLocalPart(String email) {
+        if (isEmailEmpty(email)) {
+            return null;
+        }
+        return email.substring(0, email.indexOf('@'));
+    }
+
+    private static String getEmailDomain(String email) {
+        if (isEmailEmpty(email)) {
+            return null;
+        }
+        return email.substring(email.lastIndexOf('@') + 1);
+    }
+
+    private static String getEmailDomainFromPoint(String email) {
+        if (isEmailEmpty(email)) {
+            return null;
+        }
+        return email.substring(email.indexOf(".") + 1);
+    }
+
+    public static boolean isEmailEmpty(String email) {
+        if (email == null || email.isEmpty()) {
+            LOGGER.warn("Email is empty");
+            return true;
+        }
+        return false;
+    }
+
+    public static boolean isEmailContainEt(String email) {
+        if (isEmailEmpty(email)) {
+            return false;
+        } else if (!email.contains("@")) {
+            LOGGER.warn("The email do not contains '@' character");
+            return false;
+        }
+        return true;
+    }
+
+    public static boolean isEmailIsNotStartsFromDot(String email) {
+        if (isEmailEmpty(email)) {
+            return false;
+        } else if (email.indexOf(0) == '.') {
```

Comment:
all part that are at leas twice should be constant '.' "@"
Magic number 256? 
add contant = 256